flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.js-meta.xml

```diff
@@ -10,15 +10,19 @@
     </targets>
     <targetConfigs>
         <targetConfig targets="lightning__RecordPage">
-            <property name="fields" type="String"></property>
-            <property name="mode" type="String" datasource="View,Edit"></property>
+            <property name="fields" type="String"/>
+            <property name="mode" type="String" datasource="View,Edit"/>
         </targetConfig>
         <targetConfig targets="lightning__FlowScreen">
-            <property name="recordId" type="String"></property>
-            <property name="recordTypeId" type="String"></property>
-            <property name="objectApiName" type="String"></property>
-            <property name="fields" type="String"></property>
-            <property name="mode" type="String"></property>
+            <property name="recordId" type="String" required="false" label="Record Id" description="Inout - This is the ID of the record to display or Output - The ID of a newly created record."/>
+            <property name="recordTypeId" type="String" role="inputOnly" label="Record Type Id" description="When provided, picklist values will be filter for the selected record type."/>
+            <property name="objectApiName" type="String" role="inputOnly" label="Object API Name" description="If you pass in no recordId, an Object API Name, and mode of Edit, it will open in edit mode and create a new record on save."/>
+            <property name="fields" type="String" role="inputOnly" label="Record Field List" description="A comma-separated list of field names from the recordId’s object type. If this is not provided, this component will display all fields accessible to the running user’s profile"/>
+            <property name="mode" type="String" role="inputOnly" label="Mode (View or Edit)"/>
+            <property name="flowNavigationOnSave" type="Boolean" role="inputOnly" label="Flow - Navigation on Save?" description="Check if you would like the flow to navigate to the Next screen (or Finish) when clicking Save.  For best results, hide the page footer in the flow screen."/>
```

Comment:
I like to use "Transition" to describe automatic transitions and "Navigation" to describe situations where there's some choice available. This feels a little more like a TransitionOnSave

Reply:
Would you leave flowNavigationOnCancel as is since there will be a selection made for which direction?  If so, would it be more confusing to rename the Save to Transition while the Cancel is Navigation?